### PR TITLE
Trdt 581 nino key

### PR DESCRIPTION
**Pull request by deteohm**

### CODE REVIEW

File: app/uk/gov/hmrc/transactionalrisking/controllers/GenerateReportController.scala [hunk 1]
```diff
@@ -115,8 +115,9 @@ class GenerateReportController @Inject()(
     )
   }
 
-  private def toId(rawId: String): Option[UUID] =
+  private def toId(rawId: String): Option[UUID] = {
```
Comment (basotia) [inline app/uk/gov/hmrc/transactionalrisking/controllers/GenerateReportController.scala:118]: we don't need these brackets

File: test/uk/gov/hmrc/transactionalrisking/v1/controllers/AuthorisedControllerSpec.scala [hunk 2]
```diff
@@ -83,16 +83,16 @@ class AuthorisedControllerSpec extends ControllerBaseSpec {
 
         returnedError shouldBe ninoError
 
+
       }
     }
   }
 
-//TODO uncomment me and fix me, issue might be because itmp_birthdate was removed from enrollmentauthservice
```
Comment (basotia) [inline test/uk/gov/hmrc/transactionalrisking/v1/controllers/AuthorisedControllerSpec.scala:90]: we need the TODO comment, as they might require a code fix.

File: test/uk/gov/hmrc/transactionalrisking/v1/controllers/AuthorisedControllerSpec.scala [hunk 3]
```diff
@@ -83,16 +83,16 @@ class AuthorisedControllerSpec extends ControllerBaseSpec {
 
         returnedError shouldBe ninoError
 
+
```
Comment (basotia) [inline test/uk/gov/hmrc/transactionalrisking/v1/controllers/AuthorisedControllerSpec.scala:86]: we can remove this or any other gaps in between code

File: app/uk/gov/hmrc/transactionalrisking/controllers/GenerateReportController.scala (hunk 1 above)
Comment (deteohm) [reply app/uk/gov/hmrc/transactionalrisking/controllers/GenerateReportController.scala:118]: Done

File: test/uk/gov/hmrc/transactionalrisking/v1/controllers/AuthorisedControllerSpec.scala (hunk 2 above)
Comment (deteohm) [reply test/uk/gov/hmrc/transactionalrisking/v1/controllers/AuthorisedControllerSpec.scala:90]: Checked it works in local.  Might have problems in dev/qa/et

**Pull request closed without merging**